src/tests/Properties/AssemblyInfo.cs

```diff
@@ -10,7 +10,7 @@
 [assembly: AssemblyConfiguration("")]
 [assembly: AssemblyCompany("")]
 [assembly: AssemblyProduct("tests")]
-[assembly: AssemblyCopyright("Copyright ©  2016")]
+[assembly: AssemblyCopyright("Copyright ©  2017")]
```

Comment:
I'm in favor of deleting AssemblyInfo.cs from all test projects since binaries aren't going anywhere. Either way.

Reply:
I think that's probably a good plan - but I don't like touching anything with copyrights on! 😆 I figure we'll probably update these all more generally for .NET Foundation pretty soon, so I'll leave as is for now. Thanks for looking things over!